### PR TITLE
[IA-3380] Images on Analyses tab missing a11y text.

### PR DESCRIPTION
**Pull request by n8landolt**
## Description of Ticket:
Images on the landing page for the new analysis tab are missing alt text.

## Description of Solution:
Add alt text to decorative images.

### CODE REVIEW

File: src/pages/workspaces/workspace/Analyses.js [hunk 1]
```diff
@@ -334,11 +334,9 @@ const Analyses = _.flow(
...
     div({ style: { fontSize: 48 } }, ['A place for all your analyses ']),
     div({ style: { display: 'flex', flexDirection: 'row', justifyContent: 'space-evenly', alignItems: 'center', columnGap: '5rem' } }, _.dropRight(!!googleProject ? 0 : 2, [
-      img({ src: jupyterLogo, style: { height: 120, width: 80 } }),
-      img({ src: rstudioBioLogo, style: { width: 400 } }),
-      div([
-        img({ src: galaxyLogo, style: { height: 60, width: 208 } })
-      ])
+      img({ src: jupyterLogo, style: { height: 120, width: 80 }, alt: 'Jupyter' }),
+      img({ src: rstudioBioLogo, style: { width: 400 }, alt: 'RStudio Bioconductor' }),
+      img({ src: galaxyLogo, style: { height: 60, width: 208 }, alt: 'Galaxy' })
```
Comment (cahrens) [inline src/pages/workspaces/workspace/Analyses.js:339]: If you haven't already, check with @jdcanas about why he wrapped the Galaxy image in a div.

**Review by cahrens — APPROVED** [review]

File: src/pages/workspaces/workspace/Analyses.js (hunk 1 above)
Comment (n8landolt) [reply src/pages/workspaces/workspace/Analyses.js:339]: Good idea - thank you! 
I removed it and saw it render the same, so I'm curious what his thoughts are.

Comment (n8landolt) [reply src/pages/workspaces/workspace/Analyses.js:339]: Seems like it was leftover from a different idea for positioning. As long as it looks fine without it, we're good to go!

**Review by cstrandw — APPROVED** [review]
LGTM

**Review by gpcarr — APPROVED** [review]

**Pull request merged**